jcli/src/jcli_lib/transaction/simplified.rs

```diff
@@ -0,0 +1,145 @@
+use crate::jcli_lib::rest::RestArgs;
+use crate::jcli_lib::transaction::{common, Error};
+use chain_crypto::{Ed25519, Ed25519Extended, PublicKey, SecretKey};
+
+use crate::transaction::mk_witness::WitnessType;
+use crate::transaction::staging::Staging;
+use crate::utils::key_parser::read_ed25519_secret_key_from_file;
+use crate::utils::AccountId;
+use crate::{rest, transaction};
+use chain_addr::Kind;
+use chain_core::property::FromStr;
+use chain_impl_mockchain::account::SpendingCounter;
+use chain_impl_mockchain::key::EitherEd25519SecretKey;
+use chain_impl_mockchain::transaction::Output;
+use jormungandr_lib::interfaces;
+use rand::rngs::OsRng;
+use rand::SeedableRng;
+use rand_chacha::ChaChaRng;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub struct MakeTransaction {
+    /// the account to debit the funds from
+    #[structopt(name = "ACCOUNT")]
+    pub sender_account: interfaces::Address,
+
+    /// the value
+    #[structopt(name = "VALUE")]
+    pub value: interfaces::Value,
+
+    pub block0_hash: String,
+
+    /// the file path to the file to read the signing key from.
+    /// If omitted it will be read from the standard input.
+    pub secret: Option<PathBuf>,
+
+    /// Set the change in the given address
+    pub change: Option<interfaces::Address>,
+
+    #[structopt(flatten)]
+    pub common: common::CommonTransaction,
+
+    #[structopt(flatten)]
+    pub fee: common::CommonFees,
+
+    #[structopt(flatten)]
+    rest_args: RestArgs,
```

Comment:
They need to be removed since we don't do transaction sending here anymore.

Reply:
It is still needed to query some account data.

Reply:
Oopsie, sorry.